networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/DataColumnSidecarsByRangeListenerValidatingProxy.java

```diff
@@ -60,6 +62,16 @@ public DataColumnSidecarsByRangeListenerValidatingProxy(
     this.dataColumnSidecarInclusionProofVerificationTimeSeconds =
         DATA_COLUMN_SIDECAR_INCLUSION_PROOF_VERIFICATION_HISTOGRAM.apply(
             metricsSystem, timeProvider);
+    this.dataColumnSidecarKzgBatchVerificationTimeSeconds =
+        new MetricsHistogram(
+            metricsSystem,
+            timeProvider,
+            TekuMetricCategory.BEACON,
+            "kzg_verification_data_column_batch_seconds",
```

Comment:
If you need one metric in several places, please define it only once and reuse, see https://github.com/Consensys/teku/blob/9cd9e8a373dbcf9d7f9ebc5b74c6bfa7f7c4bacd/ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/DataColumnSidecarGossipValidator.java#L56